src/Boost.sol

```diff
@@ -4,14 +4,15 @@ pragma solidity ^0.8.15;
 import "openzeppelin-contracts/access/Ownable.sol";
 import "openzeppelin-contracts/utils/cryptography/SignatureChecker.sol";
 import "openzeppelin-contracts/utils/cryptography/draft-EIP712.sol";
+import "openzeppelin-contracts/token/ERC721/extensions/ERC721URIStorage.sol";
 
 import "./IBoost.sol";
 
-contract Boost is IBoost, EIP712("boost", "1"), Ownable {
+contract Boost is IBoost, EIP712, Ownable, ERC721URIStorage {
     bytes32 public immutable eip712ClaimStructHash =
         keccak256("Claim(uint256 boostId,address recipient,uint256 amount,bytes32 ref)");
 
-    uint256 public nextBoostId = 1;
+    uint256 public nextBoostId;
```

Comment:
what's the reasoning behind this change?

Reply:
was just odd to start boost Id from 1 instead of 0

Reply:
the gas optimizoor